Character/Classes/Sorcerer/Sorcerer.xml

```diff
@@ -147,19 +147,21 @@
 				<text>At 3rd level, you gain the ability to twist your spells to suit your needs. You gain two of the following Metamagic options of your choice. You gain another one at 10th and 17th level.</text>
 				<text>	You can use only one Metamagic option on a spell when you cast it, unless otherwise noted.</text>
 			</feature>
+		</autolevel>
+		<autolevel level="3" IL="IL">
 			<feature optional="YES">
 				<name>Metamagic: Careful Spell</name>
 				<text>When you cast a spell that forces other creatures to make a saving throw, you can protect some of those creatures from the spell's full force. To do so, you spend 1 sorcery point and choose a number of those creatures up to your Charisma modifier (minimum of one creature). A chosen creature automatically succeeds on its saving throw against the spell.</text>
 			</feature>
 			<feature optional="YES">
 				<name>Metamagic: Distant Spell</name>
 				<text>When you cast a spell that has a range of 5 feet or greater, you can spend 1 sorcery point to double the range of the spell.</text>
-				<text>	When you cast a spell that has a range of touch, you can spend 1 sorcery point to make the range of the spell 30 feet.</text>
+				<text>When you cast a spell that has a range of touch, you can spend 1 sorcery point to make the range of the spell 30 feet.</text>
 			</feature>
 			<feature optional="YES">
 				<name>Metamagic: Empowered Spell</name>
 				<text>When you roll damage for a spell, you can spend 1 sorcery point to reroll a number of the damage dice up to your Charisma modifier (minimum of one). You must use the new rolls.</text>
-				<text>	You can use Empowered Spell even if you have already used a different Metamagic option during the casting of the spell.</text>
```

Comment:
Same as Distant spell metamagic, space is intentional, should remain and be added to the PS item